src/content/en/2019/http2.md

```diff
@@ -136,33 +167,37 @@ Some of this will be non-HTTPS traffic that would use HTTP/1.1 even if the serve
 
-HTTP/2 push has a mixed history despite being a much-hyped new feature of HTTP/2. The other features were basically under the hood performance improvements, but push was a brand-new concept that completely broke the single request to single response nature of HTTP up until then. It allowed extra responses to be returned: when you asked for the web page, the server could respond with the HTML page as usual, but then also send you the critical CSS and JavaScript, thus avoiding any additional round trips for certain resources. It would in theory allow us to stop inlining CSS and JavaScript into our HTML and yet still get the same performance gains of doing it. After solving that, it could potentially lead to all sorts of new and interesting use cases.
+## HTTP/2 Push
+HTTP/2 push has a mixed history despite being a much-hyped new feature of HTTP/2. The other features were basically under the hood performance improvements, but push was a brand-new concept that completely broke the single request to single response nature of HTTP up until then. It allowed extra responses to be returned: when you asked for the web page, the server could respond with the HTML page as usual, but then also send you the critical CSS and JavaScript, thus avoiding any additional round trips for certain resources. It would, in theory, allow us to stop inlining CSS and JavaScript into our HTML, and still get the same performance gains of doing so. After solving that, it could potentially lead to all sorts of new and interesting use cases.
 
-The reality has been... well, a bit disappointing. HTTP/2 push has proved much harder than originally envisaged to use effectively. Some of this has been due to [the complexity of how HTTP/2 push works](https://jakearchibald.com/2017/h2-push-tougher-than-i-thought/), and the implementation issues due to that. A bigger concern is that push can quite easily cause, rather than solve, performance issues. Over-pushing is a real risk. Often the browser is in the best place to decide *what* to request, and just as crucially *when* to request it but HTTP/2 push puts that responsibility on the server. Pushing resources that a browser already has in its cache, is a waste of bandwidth (though in my opinion so is inlining CSS but that gets must less of a hard time about that than HTTP/2 push!). [Proposals to inform the server about the status of the browser cache have stalled](https://lists.w3.org/Archives/Public/ietf-http-wg/2019JanMar/0033.html) especially on privacy concerns. Even without that problem, there are other potential issues if push is not used correctly. For example, pushing large images and therefore holding up the sending of critical CSS and JavaScript will lead to slower websites than if you'd not pushed at all!
```

Comment:
Cut out a large proportion of text here (from "Often the browser.." to "...pushed at all!"). Was that intentional?

Reply:
Yes was just trying to trim down the chapter where possible as it goes in to a lot more detail than a lot of the others I've seen, which is great! But may appear unbalanced as a whole if one chapter is a lot more of a deep-dive than others? There were also some sections which went into a lot of detail around the context of HTTP/2 that the emphasis was on that rather than the data itself.

Feel free to add back in things I've cut, but I'd recommend trimming them down if possible. Just my opinion, but would be curious to get yours and @rviscomi 's thoughts on the overall chapter balance issue 👍

Reply:
Fair point. Will leave this here for others to commit then we can accept what you have if no further comments. And yeah it definitely is a wordier chapter than some of the others! Can't stop myself sometimes 😊

Reply:
IMO the extra detail is great. The minimum expectation is to talk about the stats, but "the state of X" (for each chapter about X) also encompasses a lot of other qualitative context of X that would be nice to capture in prose. It does introduce some inconsistency with other chapters but that's ok. Just my opinion but I'll defer to what everyone else feels strongly about.